main.js

```diff
@@ -0,0 +1,57 @@
+const name = 'shaheer';
+const age = 19;
+const rating = 9.43;
+const bio = true;
+const x = null;
+const y = undefined;
+let z;
+
+const todos = [
+	{
+		id: 1,
+		text='Take out Dog'
+		iscompleted: true,
+	},
+	{
+		id: 2,
+		text='Wake up at 6'
+		iscompleted: true
+	},
+	{
+		id: 3,
+		text='Go to Gym'
+		iscompleted: true
+	},
+]
+
+for (let i = 0; i <= 10; i++) {
+	console.log('for loop number: ${i}');
+}
+
+const addnums = num1 => num1 + 5;
+console.log(addnumms(5));
+
+function person(firstname, lastname, dob) {
+	this.firstname = firstname;
+	this.lastname = lastname;
+	this.dob = new date(dob);
+}
+
+cont person1 = new person('shaheer', 'Yasin', '08-11-2004');
+
+console.log(person1.dob);
+
+//class
+class person {
+	constructor(firstname, lastname, dob) {
+		this.firstname = firstname;
+		this.lastname = lastname;
+		this.dob = new date(dob);
+	}
+	getbirthyear() {
+		return this.dob.getfullyear();
```

Comment:
JavaScript is a case-sensitive language, this method will not work.  The appropriate method to employ is 'getFullYear', which adheres to the camelCase convention, consistent with JavaScript's style guide. 

 It's worth noting that even established JavaScript methods and classes like Date maintain this convention.

For your own work, striving for consistency by emulating camelCase and PascalCase ensures a streamlined codebase. This promotes readability and adheres to best practices within the JavaScript community.

```suggestion
		return this.dob.getFullYear();
```

Reply:
Most of these issues will be resolved if you setup the VSCode properly to provide intellisense. @shaheeryyasin